restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/ClientsDialAnswerDelayTest.java

```diff
@@ -994,23 +1015,36 @@ public synchronized void testClientWithHostedApplication() throws InterruptedExc
 
     @Deployment(name = "ClientsDialAnswerDelayTest", managed = true, testable = false)
     public static WebArchive createWebArchiveNoGw() {
-        WebArchive archive = ShrinkWrap.create(WebArchive.class, "restcomm.war");
-        final WebArchive restcommArchive = ShrinkWrapMaven.resolver()
-                .resolve("org.restcomm:restcomm-connect.application:war:" + version).withoutTransitivity()
-                .asSingle(WebArchive.class);
-        archive = archive.merge(restcommArchive);
-        archive.delete("/WEB-INF/sip.xml");
-        archive.delete("/WEB-INF/conf/restcomm.xml");
-        archive.delete("/WEB-INF/data/hsql/restcomm.script");
-        archive.addAsWebInfResource("sip.xml");
-        archive.addAsWebInfResource("restcomm-delay.xml", "conf/restcomm.xml");
-        archive.addAsWebInfResource("restcomm.script_dialTest", "data/hsql/restcomm.script");
-        archive.addAsWebResource("dial-conference-entry.xml");
-        archive.addAsWebResource("dial-fork-entry.xml");
-        archive.addAsWebResource("dial-uri-entry.xml");
-        archive.addAsWebResource("dial-client-entry.xml");
-        archive.addAsWebResource("dial-number-entry.xml");
-        return archive;
-    }
+
+        Map<String,String> replacements = new HashMap();
+        //replace mediaport 2727 
+        replacements.put("2727", String.valueOf(mediaPort)); 
+        replacements.put("8080", String.valueOf(restcommHTTPPort));
+        replacements.put("8090", String.valueOf(mockPort));
+        replacements.put("5080", String.valueOf(restcommPort));
+        replacements.put("5070", String.valueOf(georgePort));        
+        replacements.put("5090", String.valueOf(bobPort));
```

Comment:
i dont get why we are replacing client ports, they dont exist in restcomm.xml, or maybe i am missing something

Reply:
for some reason the restcom conf uses some of this client ports as outboundProxy....

In addition, we use same this map to replace restcomm conf, db script, and pontentially RCML scripts, so its better to put all the ports there,whether they are goign to effectively change some file, IMO